sdk/redisenterprise/azure-resourcemanager-redisenterprise/pom.xml

```diff
@@ -48,6 +48,11 @@
     <revapi.skip>true</revapi.skip>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-json</artifactId>
+      <version>1.3.0</version> <!-- {x-version-update;com.azure:azure-json;dependency} -->
+    </dependency>
```

Comment:
probably also these (I am not sure if Alan included this change too in codegen).

4.1.38 is released but I haven't got time to test.